omero/sysadmins/server-upgrade.txt

```diff
@@ -373,8 +368,8 @@ OMERO.web migrating from Apache to Nginx
 
 Since OMERO 5.2.6 support for Apache and mod_wsgi deployment is deprecated.
 It is recommended to use :doc:`/sysadmins/unix/install-web/install-nginx` or
-:doc:`OMERO.web trial deployment </sysadmins/unix/install-web/install-web-trial>`.
-Official support is likely to be dropped during the 5.3.x line.
+:doc:`OMERO.web deployment </sysadmins/unix/install-web/web-deployment>`.
```

Comment:
This link should be updated to use the page title i.e. remove the `OMERO.web deployment` and the `<>`